test/e2e/fixture/scylla/registry.go

```diff
@@ -7,7 +7,7 @@ import (
 	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
 	scyllav1alpha1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1"
 	"github.com/scylladb/scylla-operator/pkg/assets"
-	"github.com/scylladb/scylla-operator/test/e2e/scheme"
+	"github.com/scylladb/scylla-operator/pkg/scheme"
```

Comment:
Every binary has it's own scheme - they may differ depending on which things it needs to decode. Some of the test schemes should not be in the production.

Reply:
ok, enabled strict decoding in the test scheme instead